include/triton-shared/Conversion/TritonArithToLinalg/ConversionPatterns.hpp

```diff
@@ -843,6 +850,95 @@ struct BitcastConverter : public OpConversionPattern<triton::BitcastOp> {
   }
 };
 
+struct FpToFpConverter : public OpConversionPattern<triton::FpToFpOp> {
+  using OpConversionPattern<triton::FpToFpOp>::OpConversionPattern;
+
+  LogicalResult
+  matchAndRewrite(triton::FpToFpOp op, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    auto roundingMode = triton::RoundingMode::RTNE; // default
+
+    if (auto roundingModeAttr = op->getAttrOfType<triton::RoundingModeAttr>("rounding")) {
```

Comment:
just out of curiosity, is there a way we can get the attribute without having to use the hardcoded string "rounding"?

Reply:
thank you for pointing out, I was under impression that this is how it is done because we have multiple instances in this codebase. However, tablegen makes accessor methods and there is no need to lookup strings, fixed now!

Reply:
sweet, much cleaner. thank you!